src/main/java/core/di/factory/BeanFactory.java

```diff
@@ -59,4 +63,16 @@ private Object injectBean(Class<?> injectClass) {
         return obj;
 
     }
+
+    private boolean existInjectConstructor(Class findClass) {
+        return Optional.ofNullable(BeanFactoryUtils.getInjectedConstructor(findClass)).isPresent();
```

Comment:
이전에 말씀드렸던 의미는 아래의 코드를 의도했었어요
```suggestion
        return BeanFactoryUtils.getInjectedConstructor(findClass) != null;
```
더 나아가 hasInjectedConstructor 메서드를 만들어 보는 것도 도움이 될 것 같아요